File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopSearchResponseParam.java
package com.cakk.domain.mysql.dto.param.shop;

import java.util.Set;

public record CakeShopSearchResponseParam(
	Long cakeShopId,
	String thumbnailUrl,
	String cakeShopName,
	String cakeShopBio,
	Set<String> cakeImageUrls,
	Set<CakeShopOperationParam> operationDays
) {

}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/shop/CakeShopSearchResponseParam.kt
package com.cakk.domain.mysql.dto.param.shop

data class CakeShopSearchResponseParam(
	val cakeShopId: Long,
	val thumbnailUrl: String,
	val cakeShopName: String,
	val cakeShopBio: String,
	val cakeImageUrls: Set<String>,
	val operationDays: Set<CakeShopOperationParam>
)
